### DISCUSSION TITLE
Categories

### DISCUSSION THREAD

**Discussion by TonySLondon** [moneymanagerex/moneymanagerex#4858] [Ideas]
I don't like the new category display, although I am sure many will. I preferred the previous structure, where the categories were divided into headings. Would it be possible to have this as an option for display?

**Comment by vomikan** [comment]
@TonySLondon I recommend to press the Enter button when the cursor on the category combobox field.

**Comment by vomikan** [comment]
@whalley I think an easy fix is call the categories dialog if the combobox field is empty and dropdown event happens.
Update: It turned out to be harder than I thought.

**Reply by whalley** [reply]
@vomikan If there is a strong appetite for the old dialog then perhaps we need to add another button to select categories alongside the combobox and split transactions button! But now the UI gets messy with all these options. Personally, I suggest we leave it as-is, people will get familiar with it.

**Comment by TonySLondon** [comment]
Thank you Nikolay, that works.

------ Original Message ------
From "Nikolay" ***@***.***>
To "moneymanagerex/moneymanagerex" ***@***.***>
Cc "Tony Speigal" ***@***.***>; "Mention" 
***@***.***>
Date 23/07/2022 16:55:17
Subject Re: [moneymanagerex/moneymanagerex] Categories (Discussion 
#4858)

>
***@***.*** <https://github.com/TonySLondon> I recommend to press the 
>Enter button when the cursor on the category combobox field.
>
>—
>Reply to this email directly, view it on GitHub 
><https://github.com/moneymanagerex/moneymanagerex/discussions/4858#discussioncomment-3212931>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AVAHBZJPBYMTI6FZ3QQJ2RTVVQIWLANCNFSM54OACAGQ>.
>You are receiving this because you were mentioned.Message ID: 
***@***.***>
>

**Comment by TonySLondon** [comment]
Thanks Whalley, that is a great help. Just what I needed.
------ Original Message ------
From "Nikolay" ***@***.***>
To "moneymanagerex/moneymanagerex" ***@***.***>
Cc "Tony Speigal" ***@***.***>; "Mention" 
***@***.***>
Date 23/07/2022 16:57:32
Subject Re: [moneymanagerex/moneymanagerex] Categories (Discussion 
#4858)

>
***@***.*** <https://github.com/whalley> I think an easy fix is call the 
>categories dialog if the combobox field is empty and dropdown event 
>happens.
>
>—
>Reply to this email directly, view it on GitHub 
><https://github.com/moneymanagerex/moneymanagerex/discussions/4858#discussioncomment-3212936>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AVAHBZJF4RZX4Q7U5VT3PPDVVQI6ZANCNFSM54OACAGQ>.
>You are receiving this because you were mentioned.Message ID: 
***@***.***>
>
